### PR TITLE
Removed duplicate call to check for bedrock file that enforces project commands order

### LINKED ISSUE #927: Remove duplicate bedrock file check and merge constants files
**As a:**
Developer

**I want:**
-  [ ] Remove duplicate check for bedrock file in `project pipeline` command

https://github.com/CatalystCode/spk/blob/2bbf56fec331f38fb6932eada39fe4cdd9e86caa/src/commands/project/pipeline.ts#L116

https://github.com/CatalystCode/spk/blob/2bbf56fec331f38fb6932eada39fe4cdd9e86caa/src/commands/project/pipeline.ts#L45

 - [ ] Merge `contants.ts` files

https://github.com/CatalystCode/spk/blob/master/src/lib/constants.ts

https://github.com/CatalystCode/spk/blob/master/src/constants.ts

**Acceptance Criteria:**
Implement above tasks.

**Does this require updates to documentation?:**
No

### PR DESCRIPTION
**Pull request by sarath-p**
- Closes https://github.com/microsoft/bedrock/issues/927

- Also merged `constants.ts` files.

### CODE REVIEW

File: src/lib/bedrockYaml.ts [hunk 1]
```diff
@@ -104,3 +106,38 @@ export const addNewService = (
   });
   fs.writeFileSync(path.join(absPath, YAML_NAME), asYaml);
 };
+
+/**
+ * Returns bedrock file information
+ *
+ * @param rootProjectPath Path to read the bedrock.yaml file
+ */
+export const fileInfo = async (
```
Comment (dennisseah) [inline src/lib/bedrockYaml.ts]: need not be an `async` function

File: src/lib/bedrockYaml.ts [hunk 2]
```diff
@@ -104,3 +106,38 @@ export const addNewService = (
...
+    throw new Error("Project root path is not valid");
+  }
+
+  const absProjectPath = path.resolve(rootProjectPath);
+
+  let bedrockFile: IBedrockFile | undefined;
+
+  try {
+    bedrockFile = read(absProjectPath);
+    logger.debug(
```
Comment (dennisseah) [inline src/lib/bedrockYaml.ts:120]: can be better written as
```
const bedrockFile = read(absProjectPath);
```

and remove
```
let bedrockFile: IBedrockFile | undefined;
```
avoiding the `undefined`

**Review by dennisseah — CHANGES_REQUESTED** [review]

**Review by dennisseah — APPROVED** [review]
LGTM

**Review by dennisseah — APPROVED** [review]

File: src/lib/constants.ts [hunk 3]
```diff
@@ -1,2 +1,11 @@
 export const BUILD_SCRIPT_URL =
   "https://raw.githubusercontent.com/Microsoft/bedrock/master/gitops/azure-devops/build.sh";
+
+export const projectInitDependencyErrorMessage =
```
Comment (mtarng) [inline src/lib/constants.ts]: Can these variable names be changed to match the all uppercase convention?

ie. PROJECT_INIT_DEPENDENCY_ERROR_MESSAGE.

or something similar?

**Review by mtarng — CHANGES_REQUESTED** [review]

**Review by mtarng — APPROVED** [review]

**Pull request merged**